### ISSUE TITLE
Reusing the config file on install skips authorization

### ISSUE THREAD

**Issue by frisco82** [alexa-pi/AlexaPi#186]
Thank you for reporting an issue with **AlexaPi**.

Make sure you are running the latest version of **AlexaPi**, and look through [issues](https://github.com/alexa-pi/AlexaPi/issues?utf8=%E2%9C%93&q=is%3Aissue) before submitting.

Check out our [Wiki](https://github.com/alexa-pi/AlexaPi/wiki/), and make sure to look at [Debugging](https://github.com/alexa-pi/AlexaPi/wiki/Debugging/), and [Audio debugging](https://github.com/alexa-pi/AlexaPi/wiki/Audio-setup-&-debugging) in particular before filing an issue.

In order to help troubleshooting, be sure to include the following information:

**Output of the [audio debugging script](https://github.com/alexa-pi/AlexaPi/wiki/Audio-setup-&-debugging#audio-debugging-script).**

**Your OS (including version) where you are running AlexaPi:**
```
OSMC (Debian 8.7)
```

**Your hardware platform and model you are running on:**
```
Raspberry PI 2
```

**Python release (`python2 --version`):**
```
Python 2.7.9
```

**Description of problem:**
```
If you already have a config file the installer will ask if you want to Keep, Edit or Delete, choosing Keep it does exit the installer even if there is no token saved in the config file. You need to run manually `python auth_web.py` to finish the configuration.
```

**Expected:**
```
Check if there is a token and is not expired and only then skip the authorization part.
```

**Problem-relevant `config.yaml` entries:**
```

```

**Steps to reproduce:**

1. Install AlexaPi and kill or get an error after config is created but before authorization is saved 
2. Try to install again

**Traceback (if applicable):**
```

```

**Additional info:**
```

```

**Comment by renekliment** [comment]
Yeah, this makes sense. Do you wanna do a PR? This should be relatively easy.
